### ISSUE TITLE
Problem with token issuer validation

### ISSUE THREAD

**Issue by AlbertoPa** [1iveowl/custom-claim-provider#1]
Hello, and than you for releasing this sample, which is of great help!

I get a token issuer validation error:

`Microsoft.IdentityModel.Tokens.SecurityTokenInvalidIssuerException: IDX40001: Issuer: 'https://{{tenantId}}.ciamlogin.com/{{tenantId}}/v2.0', does not match any of the valid issuers provided for this application.`

when I use the format in the README for authority and instance, which uses

`https://login.microsoftonline.com/{{tenantId}}/v2.0`

and

`https://login.microsoftonline.com/`

Did the format change?

**Comment by 1iveowl** [comment]
Hi @AlbertoPa are you still seeing this issue? 

I will need to run a test to see if anything changed.

**Comment by AlbertoPa** [comment]
Yes, if I use "login.microsoftonline.com..." as Authority (Instance is fine) with Entra ID for Customers

I need to use: 

```
        "Authority": "https://{tenantName}.ciamlogin.com/{tenantId}/v2.0",
        "Instance": "https://login.microsoftonline.com/",
```

I hope this helps :-)

**Comment by 1iveowl** [comment]
The sample have been updated and tested. 

With the Entra External ID experience and newer unified app registration model, Microsoft replaced this property with `requestedAccessTokenVersion` to align with modern configuration patterns and improve clarity.

Also, the readme have been improved.
